# Licensed to the Apache Software Foundation (ASF) under one
# or more contributor license agreements.  See the NOTICE file
# distributed with this work for additional information
# regarding copyright ownership.  The ASF licenses this file
# to you under the Apache License, Version 2.0 (the
# "License"); you may not use this file except in compliance
# with the License.  You may obtain a copy of the License at
#
#   http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing,
# software distributed under the License is distributed on an
# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
# KIND, either express or implied.  See the License for the
# specific language governing permissions and limitations
# under the License.
#
# This patch file was generated by NetBeans IDE
# It uses platform neutral UTF-8 encoding and \n newlines.
diff --git a/graal-js/mx.graal-js/suite.py b/graal-js/mx.graal-js/suite.py
index 0ddd2490be..dc165e33b9 100644
--- a/graal-js/mx.graal-js/suite.py
+++ b/graal-js/mx.graal-js/suite.py
@@ -254,9 +254,6 @@ suite = {
     "com.oracle.js.parser" : {
       "subDir" : "src",
       "sourceDirs" : ["src"],
-      "dependencies" : [
-        "sdk:GRAAL_SDK"
-      ],
       "exports" : [],
       "javaCompliance" : "1.8",
       "checkstyleVersion" : "8.8",
@@ -454,6 +451,19 @@ suite = {
   },
 
   "distributions" : {
+    "GRAAL_JS_PARSER" : {
+      "moduleName" : "com.oracle.js.parser",
+      "subDir" : "src",
+      "dependencies" : ["com.oracle.js.parser"],
+      "description" : "Graal Javascript parser",
+      "distDependencies" : [
+        "truffle:TRUFFLE_API",
+      ],
+      "maven" : {
+        "artifactId" : "js-parser",
+      },
+      "license":"UPL"
+    },
     "GRAALJS" : {
       "moduleName" : "org.graalvm.js",
       "subDir" : "src",
